senator/__main__.py

```diff
@@ -1,8 +1,15 @@
 from senator.utility import *
 from senator.game import *
 
-votes, initial_utility = parse_data(load_data(FILE_PATH))
+OWNER_LOSS = 0.59
+OTHER_LOSS = 0.19
+OWNER_TRASHOLD = 0.18 #Set 2.0 to change the model TODO: find better solution than 2.0
 
-game = Game(initial_utility,votes)
 
-game.run_greedy_search()
\ No newline at end of file
+votes, initial_utility, is_owner = parse_data(load_data(FILE_PATH))
+game = Game(initial_utility, votes, is_owner, OWNER_LOSS, OTHER_LOSS, OWNER_TRASHOLD)
+#game = Game(np.zeros((3,17)),votes)
+
+actions_in_steps, final_utility, strategies = game.run_greedy_search()
+print(strategies)
+#print(final_utility.sum(axis=1))
```

Comment:
Veškerý kód který se volá, včetně tohohle, se většinou dává do nějaké funkce, např. `main`, a pak na konci tohohle souboru dej
 ```python
if __name__ == '__main__':
    main()
```
To je taková magická formule, která spustí funkci `main` pokud je tenhle soubor spuštěn přímo z command liny. Můžeš se zeptat chatu na to co to přesněji dělá, myslím že by to měl být schopen docela dobře vysvětlit.

Reply:
Ok, done